### PR TITLE
file correction

### PR DESCRIPTION
**Pull request by Albrecht-Albi**

### CODE REVIEW

**Comment by Albrecht-Albi** [comment]
created the initial noResults

File: src/features/MoviesList/index.js [hunk 1]
```diff
@@ -12,35 +12,37 @@ const MoviesList = () => {
...
-            </List>
-          </>
-        }
-      />
-      <Pagination />
-    </>
+    status === "loading" ?
+      <Loader /> :
+      status === "error" ?
+        <Error /> :
```
Comment (lkkolosowski) [inline src/features/MoviesList/index.js:20]: Very good conditional block, great 🔥🔥😁

**Review by lkkolosowski — APPROVED** [review]
Let's go

File: src/common/NoResults/styled.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      }
+      
+      75% {
+        transform: translate(6px, 0px);
+      }
+            
+      100% {
+        transform: translate(0, 0px);
+      }
+    }
```
Comment (lkkolosowski) [inline src/common/NoResults/styled.js:52]: WOOOW What a beatiful animation! ❤️‍🔥❤️‍🔥 

BirdParrotGIF (2)

**Pull request merged**